app/js/elements/fxos-customizer-highlighter.js

```diff
@@ -14,8 +14,23 @@ var shadowHTML =
   outline: 1px solid #00caf2;
   opacity: 0.75;
 }
+.label {
```

Comment:
It would be good to apply some ellipsizing here if the text is too long, which is definitely a possibility.